Dockerfile.ubuntu

```
@@ -5,6 +5,7 @@ WORKDIR /usr/src/app/
 COPY package.json yarn.lock ./
 COPY packages packages
 
+RUN apt-get install -yq git
```

We update the APT cache elsewhere in the Dockerfile. I'm not 100% sure if it's best practice, but at least it makes sure you're up to date.
```suggestion
RUN apt-get update && apt-get install -yq git
```